### PR TITLE
cabal-docspec

### PR DESCRIPTION
**Pull request by phadej**

### CODE REVIEW

File: src/Numeric/Log/Signed.hs [hunk 1]
```diff
@@ -31,6 +31,8 @@ import Data.Functor ((<$>))
 #endif
 
 -- $setup
+-- >>> let nxor = (==)
+-- >>> let multSign b = if b then id else (*) (-1)
```
Comment (RyanGlScott) [inline src/Numeric/Log/Signed.hs:35]: Aren't `nxor` and `multSign` defined in this module? Why are they redefined here?

Comment (phadej) [reply src/Numeric/Log/Signed.hs:35]: Because they are not exported.

Comment (RyanGlScott) [reply src/Numeric/Log/Signed.hs:35]: Ah, right. I'm a bit leery about the blatant code duplication here, but I suppose there's not much that can be done about that. I do want to make sure that if the definitions of `nxor`/`multSign` change in the future, then their doctest counterparts also get updated. In pursuit of that goal, would it be possible to either:

1. Put these doctests next to the definitions so it's more visually obvious, or
2. Leave a comment somewhere stating that the code for `nxor`/`multSign` should stay in sync with these doctests?

Comment (phadej) [reply src/Numeric/Log/Signed.hs:35]: 3. Move tests into proper test-suite ;)

Comment (phadej) [reply src/Numeric/Log/Signed.hs]: I merged `$setup` block with a test-block

**Pull request merged**